### PR TITLE
Help box

### LINKED ISSUE #132: Need for more informative help popup
The current help popup only provides a single link to the user guide. Perhaps commonly-used commands might be more helpful to the user

### PR DESCRIPTION
**Pull request by JinHao-L**
* Add commonly-used command to help message
* Update DG:
	* Add intro
	* Add details to "appendix: instructions for manual testing"
* Remove appointment ID
* Update Model class diagram (with removed appt ID)

image

closes #132

### CODE REVIEW

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 1]
```diff
@@ -16,14 +16,31 @@
 public class HelpWindow extends UiPart<Stage> {
 
     public static final String USERGUIDE_URL = "https://ay2021s1-cs2103t-t12-4.github.io/tp/UserGuide.html";
-    public static final String HELP_MESSAGE = "Refer to the user guide: " + USERGUIDE_URL;
+
+    public static final String COMMON_COMMANDS = "Add a patient ---"
+            + "add n/NAME i/NRIC p/PHONE_NUMBER a/ADDRESS [t/TAG]…\n"
+            + "Edit a patient --- edit INDEX [n/NAME] [i/NRIC] [p/PHONE_NUMBER] [a/ADDRESS] [t/TAG]…\n"
```
Comment (Avalionnet) [inline src/main/java/seedu/address/ui/HelpWindow.java]: Perhaps the prefix [r/REMARK] can be added to the command syntax for both the add and edit commands.
I've just made this change in our user guide as well in my latest PR 👍🏻

File: src/main/java/seedu/address/ui/HelpWindow.java [hunk 2]
```diff
@@ -16,14 +16,31 @@
...
+            + "Delete a patient --- delete INDEX\n"
+            + "List all patients --- list\n\n"
+            + "Assign a patient to an appointment --- assign INDEX d/DATE t/TIME [dur/DURATION]\n"
+            + "Change an appointment time --- change INDEX [d/DATE] [t/TIME] [dur/DURATION]\n"
+            + "Cancel an appointment --- cancel d/DATE t/TIME\n"
+            + "Mark an appointment as done --- done d/DATE t/TIME\n"
+            + "Find available time slots --- avail d/DATE\n"
+            + "View appointments by date --- view [d/DATE]\n\n"
+            + "Undo a previous command --- undo\n"
+            + "Redo a previous command --- redo\n";
```
Comment (Avalionnet) [inline src/main/java/seedu/address/ui/HelpWindow.java:33]: Perhaps a line showcasing the remark command can be included as well? Let me know what you think 😄

**Review by Avalionnet — COMMENTED** [review]
This is an excellent addition to Nuudle! Thank you for the outstanding work 👍🏻 
LGTM other than the 2 comments :)

**Review by xz0127 — COMMENTED** [review]
Other than the comments mentioned by Ming Lim, the rest LGTM! 👍

File: src/main/java/seedu/address/ui/HelpWindow.java (hunk 1 above)
Comment (JinHao-L) [reply src/main/java/seedu/address/ui/HelpWindow.java]: Ok I will add it in and update the helpMessage image

File: src/main/java/seedu/address/ui/HelpWindow.java (hunk 2 above)
Comment (JinHao-L) [reply src/main/java/seedu/address/ui/HelpWindow.java:33]: Ok added

**Review by xz0127 — APPROVED** [review]
LGTM! 👍

**Pull request merged**